packages/core/src/js/transports/encodePolyfill.ts

```diff
@@ -6,7 +8,7 @@ export const useEncodePolyfill = (): void => {
     (RN_GLOBAL_OBJ.__SENTRY__ as Partial<(typeof RN_GLOBAL_OBJ)['__SENTRY__']>) = {};
   }
 
-  RN_GLOBAL_OBJ.__SENTRY__.encodePolyfill = encodePolyfill;
+  RN_GLOBAL_OBJ.__SENTRY__[SDK_VERSION].encodePolyfill = encodePolyfill;
```

Comment:
The `SDK_VERSION` is created as a side effect of creating Sentry Logger (https://github.com/getsentry/sentry-javascript/blob/bf323b111b7e7b781e60e6299e432ce89f144970/packages/core/src/utils-hoist/logger.ts#L102)

But it's not guaranteed that it will exist.

We should use https://github.com/getsentry/sentry-javascript/blob/d773cb7324480ed3cffc14504f0e41951e344d19/packages/core/src/carrier.ts#L42

Reply:
Also can be a follow up PR.